pkg/pushsync/pushsync.go

```diff
@@ -276,3 +255,29 @@ func (ps *PushSync) PushChunkToClosest(ctx context.Context, ch swarm.Chunk) (*Re
 
 	return rec, nil
 }
+
+func (ps *PushSync) deliverToPSS(ctx context.Context, ch swarm.Chunk) error {
```

Comment:
because PSS uses push sync to send messages. when it receives a message it is the closest point from which we can deliver to PSS. Otherwise the chunk will land in localstore and we have to intercept there. which means that for any normal upload too the chunks will be delivered to PSS which is not good.

See this comment by @acud https://github.com/ethersphere/bee/pull/479#discussion_r466326141